### PR TITLE
[DL Edition] Changes in author information

### PR DESCRIPTION
**Pull request by PaulaKramer**
## Description
Linked NextAID in DL talktorials (see https://github.com/volkamerlab/teachopencadd/issues/392) 

## Todos
- [x] Added NextAID in author information in each DL talktorials 
- [x] Regenerated readmes
- [x] Mention new DL content in readme and on project website

## Status
- [x] Ready to go

### CODE REVIEW

**Review by AndreaVolkamer — APPROVED** [review]
@PaulaKramer thanks!

File: teachopencadd/talktorials/T033_molecular_representations/README.md
Comment (mbackenkoehler) [inline teachopencadd/talktorials/T033_molecular_representations/README.md:1]: I would leave out the "member of the" part, but that maybe is a matter of taste.

File: README.md
Comment (mbackenkoehler) [inline README.md:1]: "and _its_ application"?

File: docs/index.rst
Comment (mbackenkoehler) [inline docs/index.rst:1]: See README.md

**Comment by mbackenkoehler** [comment]
Merging this will close #392

**Pull request merged**